evm_loader/lib/src/types/tracer_ch_db.rs

```diff
@@ -244,22 +262,33 @@ impl ClickHouseDb {
         "#;
 
         let time_start = Instant::now();
-        let slot_opt = Self::row_opt(
+        let mut slot_opt = Self::row_opt(
             self.client
-                .query(query)
+                .query(query_last_day)
                 .bind(key)
                 .bind(slot)
                 .fetch_one::<u64>()
                 .await,
-        )?;
+        );
+
+        if slot_opt.is_err() {
```

Comment:
If no records were returned, this will be false.

Reply:
Yep, but the code of row_opt a little bit misleading.
I'll remove the usage of row_opt.

```Rust
    fn row_opt<T>(result: clickhouse::error::Result<T>) -> clickhouse::error::Result<Option<T>> {
        match result {
            Ok(row) => Ok(Some(row)),
            Err(clickhouse::error::Error::RowNotFound) => Ok(None),
            Err(e) => Err(e),
        }
    }
```